plugin/src/main/java/io/deephaven/plugin/type/JsType.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.plugin.type;
+
+import io.deephaven.plugin.Plugin;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Path;
+
+/**
+ * A "js type" plugin. Useful for adding custom javascript code to the server.
+ */
+public interface JsType extends Plugin {
```

Comment:
Not sure I like the name `JsType` - that's already used in GWT for identifying JS objects, why not named `JsPlugin`?

Reply:
I'm happy to rename!